kitsune/questions/tasks.py

```diff
@@ -2,14 +2,14 @@
 from datetime import date
 from typing import Dict
 
-# NOTE: This import is just so _fire_task gets registered with celery.
-import tidings.events  # noqa
 from celery import task
 from django.conf import settings
 from django.contrib.auth.models import User
 from django.db import connection, transaction
 from sentry_sdk import capture_exception
 
+# NOTE: This import is just so _fire_task gets registered with celery.
+import kitsune.tidings.events  # noqa
```

Comment:
I was wondering if it might be better to register the tidings Celery tasks in the app config? What if we change the tidings app config to something like:

```python
class TidingsConfig(AppConfig):
    name = "kitsune.tidings"
    default_auto_field = "django.db.models.AutoField"

    def ready(self):
        # Register the tidings Celery tasks.
        from kitsune.tidings import events, tasks  # noqa
```

and then move `kitsune.tidings` such that it's the first of the kitsune apps? Something we can consider later if at all, just thinking out loud.